MiniProject 1/VideoViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  VideoViewController.swift
+//  MiniProject 1
+//
+//  Created by ESIEA on 10/04/2015.
+//  Copyright (c) 2015 HaiNguyen. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+class VideoViewController: UIViewController
+{
+    var moviePlayer: AVPlayer?
+    var playerLayer: AVPlayerLayer?
+    var playerItem: AVPlayerItem?
+    let screenSize = UIScreen.mainScreen().bounds
+    var urls = ["http://fun.siz.io/stories/142893791787803c7fb48f4d/0.mp4",
+                "http://fun.siz.io/stories/142893791787803c7fb48f4d/1.mp4",
+                "http://fun.siz.io/stories/142893791787803c7fb48f4d/2.mp4",
+                "http://fun.siz.io/stories/142893791787803c7fb48f4d/3.mp4"]
```

Comment:
I forgot to tell you but it's will be better if all videos are differents (and not from the same story).